ext/net/02_tls.js

```diff
@@ -137,29 +129,96 @@ class TlsListener extends Listener {
   }
 }
 
+/**
+ * Returns true if this object has the shape of one of the certified key material
+ * interfaces.
+ */
 function hasTlsKeyPairOptions(options) {
   return (ReflectHas(options, "cert") || ReflectHas(options, "key") ||
     ReflectHas(options, "certFile") ||
-    ReflectHas(options, "keyFile"));
+    ReflectHas(options, "keyFile") || ReflectHas(options, "privateKey") ||
+    ReflectHas(options, "certChain"));
 }
```

Comment:
This function would return `true` if `options` just had the `cert` property. The same would be true for the other properties. This seems wrong as options require both `cert` and `key` properties, along with other pair combinations.

Reply:
Correct -- this function should return true if we match any part of the TLS CertifiedKey options so we can do further checking later on. Once we land support for optional cppgc objects we'll be able to remove this entirely and just return null from `loadTlsKeyPair`.